### ISSUE TITLE
Can't access to the source of external libraries if you use a Microsoft account

### ISSUE THREAD

**Issue by AndreuCodina** [scalameta/metals#4600]
### Describe the bug

If you use a Microsoft account on your computer with Windows, the operating system automatically created the name of your personal folder. For example, my folder is:

`C:\Users\Andreu Codina`

It's my name and my surname, but the important thing is there is a white space between them.

You can take any code, for example:
```
@main def main(): Unit =
  println("Hello!")
```

If you go to the definition of `println`, instead of viewing its definition, you see a string:
`C:\Users\Andreu%20Codina\AppData\Local\Coursier\cache\v1\https\repo1.maven.org\maven2\org\scala-lang\scala-library\2.13.8\scala-library-2.13.8-sources.jar`

So the white space is replaced with %20, and that path doesn't exist.

Metals server versions tested:
- 0.11.9 (the latest)
- 0.11.9+85-286899a2-SNAPSHOT
- 0.11.9+91-762cdcff-SNAPSHOT

This is the log of VSCode with Metals when I create a new project and import the build (you can see `Andreu%2520Codina`):

```
Java home: C:\Users\Andreu Codina\AppData\Local\Coursier\cache\arc\https\github.com\AdoptOpenJDK\openjdk8-binaries\releases\download\jdk8u292-b10\OpenJDK8U-jdk_x64_windows_hotspot_8u292b10.zip\jdk8u292-b10
Metals version: 0.11.9
2022.11.03 10:18:59 INFO  logging to file C:\git\scala10\.metals\metals.log
2022.11.03 10:18:59 INFO  Started: Metals version 0.11.9 in workspace 'C:\git\scala10' for client Visual Studio Code 1.72.2.
2022.11.03 10:19:00 INFO  Flyway Community Edition 9.4.0 by Redgate
2022.11.03 10:19:00 INFO  See what's new here: https://flywaydb.org/documentation/learnmore/releaseNotes#9.4.0
2022.11.03 10:19:00 INFO  
2022.11.03 10:19:00 INFO  Database: jdbc:h2:file:C:\git\scala10\.metals\metals (H2 2.1)
2022.11.03 10:19:00 INFO  Successfully validated 4 migrations (execution time 00:00.030s)
2022.11.03 10:19:00 INFO  Creating Schema History table "PUBLIC"."flyway_schema_history" ...
2022.11.03 10:19:00 INFO  Current version of schema "PUBLIC": << Empty Schema >>
2022.11.03 10:19:00 INFO  Migrating schema "PUBLIC" to version "1 - Create tables"
2022.11.03 10:19:00 INFO  Migrating schema "PUBLIC" to version "2 - Server discovery"
2022.11.03 10:19:00 INFO  Migrating schema "PUBLIC" to version "3 - Jar symbols"
2022.11.03 10:19:00 INFO  Migrating schema "PUBLIC" to version "4 - Fingerprints"
2022.11.03 10:19:00 INFO  Successfully applied 4 migrations to schema "PUBLIC", now at version v4 (execution time 00:00.075s)
2022.11.03 10:19:01 INFO  time: initialize in 1.94s
Initializing Scala Debugger
Release notes was not shown: not showing release notes since they've already been seen for your current version
2022.11.03 10:19:01 WARN  Build server is not auto-connectable.
2022.11.03 10:19:01 WARN  no build tool detected in workspace 'C:\git\scala10'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. 
2022.11.03 10:19:28 INFO  
2022.11.03 10:19:28 INFO  Template applied in C:\git\.\scala10
2022.11.03 10:19:28 INFO  
2022.11.03 10:19:28 INFO  time: ran 'giter8.Giter8' in 7.43s
2022.11.03 10:19:44 WARN  no build target for: C:\git\scala10\src\main\scala\Main.scala
2022.11.03 10:19:45 INFO  no build target found for C:\git\scala10\src\main\scala\Main.scala. Using presentation compiler with project's scala-library version: 3.2.0
2022.11.03 10:19:47 INFO  time: code lens generation in 2.15s
2022.11.03 10:19:55 INFO  running 'C:\Users\Andreu Codina\AppData\Local\Coursier\cache\arc\https\github.com\AdoptOpenJDK\openjdk8-binaries\releases\download\jdk8u292-b10\OpenJDK8U-jdk_x64_windows_hotspot_8u292b10.zip\jdk8u292-b10\bin\java -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -jar C:\Users\ANDREU~1\AppData\Local\Temp\metals8389400239816882182\sbt-launch.jar -Dbloop.export-jar-classifiers=sources bloopInstall'
2022.11.03 10:19:58 INFO  [info] welcome to sbt 1.7.1 (AdoptOpenJDK Java 1.8.0_292)
2022.11.03 10:19:59 INFO  [info] loading global plugins from C:\Users\Andreu Codina\.sbt\1.0\plugins
2022.11.03 10:20:02 INFO  [info] loading settings for project scala10-build-build from metals.sbt ...
2022.11.03 10:20:02 INFO  [info] loading project definition from C:\git\scala10\project\project
2022.11.03 10:20:05 INFO  [info] loading settings for project scala10-build from metals.sbt ...
2022.11.03 10:20:05 INFO  [info] loading project definition from C:\git\scala10\project
2022.11.03 10:20:07 INFO  [success] Generated .bloop\scala10-build.json
2022.11.03 10:20:07 INFO  [success] Total time: 2 s, completed Nov 3, 2022 10:20:07 AM
2022.11.03 10:20:08 INFO  [info] loading settings for project root from build.sbt ...
2022.11.03 10:20:08 INFO  [info] set current project to scala10 (in build file:/C:/git/scala10/)
2022.11.03 10:20:08 INFO  [success] Generated .bloop\root.json
2022.11.03 10:20:08 INFO  [success] Generated .bloop\root-test.json
2022.11.03 10:20:08 INFO  [success] Total time: 1 s, completed Nov 3, 2022 10:20:09 AM
2022.11.03 10:20:10 INFO  time: ran 'sbt bloopInstall' in 14s
2022.11.03 10:20:10 INFO  Attempting to connect to the build server...
2022.11.03 10:20:10 INFO  Bloop uses C:\Users\Andreu Codina\AppData\Local\Coursier\cache\arc\https\github.com\AdoptOpenJDK\openjdk8-binaries\releases\download\jdk8u292-b10\OpenJDK8U-jdk_x64_windows_hotspot_8u292b10.zip\jdk8u292-b10 defined at C:\Users\Andreu Codina\.bloop\bloop.json
2022.11.03 10:20:10 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\git\scala10\.metals\bsp.trace.json or C:\Users\Andreu Codina\AppData\Local\scalameta\metals\cache\bsp.trace.json
2022.11.03 10:20:10 INFO  Attempting to connect to the build server...
2022.11.03 10:20:10 INFO  Bloop uses C:\Users\Andreu Codina\AppData\Local\Coursier\cache\arc\https\github.com\AdoptOpenJDK\openjdk8-binaries\releases\download\jdk8u292-b10\OpenJDK8U-jdk_x64_windows_hotspot_8u292b10.zip\jdk8u292-b10 defined at C:\Users\Andreu Codina\.bloop\bloop.json
2022.11.03 10:20:10 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\git\scala10\project\.metals\bsp.trace.json or C:\Users\Andreu Codina\AppData\Local\scalameta\metals\cache\bsp.trace.json
2022.11.03 10:20:10 INFO  time: Connected to build server in 0.34s
2022.11.03 10:20:10 INFO  Connected to Build server: Bloop v1.5.4
2022.11.03 10:20:14 ERROR issues while parsing: C:\Users\Andreu Codina\AppData\Local\Coursier\cache\v1\https\repo1.maven.org\maven2\org\scala-lang\scala3-library_3\3.2.0\scala3-library_3-3.2.0-sources.jarjava.nio.file.InvalidPathException: Illegal char <:> at index 3: jar:file:///C:/Users/Andreu%2520Codina/AppData/Local/Coursier/cache/v1/https/repo1.maven.org/maven2/org/scala-lang/scala3-library_3/3.2.0/scala3-library_3-3.2.0-sources.jar!/scala/Tuple.scala
	at sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
	at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
	at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
	at sun.nio.fs.WindowsPath.parse(WindowsPath.java:94)
	at sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:255)
	at java.nio.file.Paths.get(Paths.java:84)
	at scala.meta.internal.mtags.CommonMtagsEnrichments$XtensionInputVirtual.$anonfun$filename$2(CommonMtagsEnrichments.scala:337)
	at scala.util.Failure.getOrElse(Try.scala:219)
	at scala.meta.internal.mtags.CommonMtagsEnrichments$XtensionInputVirtual.filename(CommonMtagsEnrichments.scala:337)
	at scala.meta.internal.mtags.ScalaToplevelMtags.loop(ScalaToplevelMtags.scala:203)
	at scala.meta.internal.mtags.ScalaToplevelMtags.indexRoot(ScalaToplevelMtags.scala:50)
	at scala.meta.internal.mtags.MtagsIndexer.index(MtagsIndexer.scala:19)
	at scala.meta.internal.mtags.MtagsIndexer.index$(MtagsIndexer.scala:18)
	at scala.meta.internal.mtags.ScalaToplevelMtags.index(ScalaToplevelMtags.scala:40)
	at scala.meta.internal.mtags.Mtags.toplevels(Mtags.scala:32)
	at scala.meta.internal.mtags.SymbolIndexBucket.indexSource(SymbolIndexBucket.scala:128)
	at scala.meta.internal.mtags.SymbolIndexBucket.addSourceFile(SymbolIndexBucket.scala:105)
	at scala.meta.internal.mtags.SymbolIndexBucket.$anonfun$addSourceJar$2(SymbolIndexBucket.scala:67)
	at scala.collection.immutable.List.flatMap(List.scala:293)
	at scala.meta.internal.mtags.SymbolIndexBucket.$anonfun$addSourceJar$1(SymbolIndexBucket.scala:65)
	at scala.meta.internal.io.PlatformFileIO$.withJarFileSystem(PlatformFileIO.scala:87)
	at scala.meta.internal.io.FileIO$.withJarFileSystem(FileIO.scala:44)
	at scala.meta.internal.mtags.SymbolIndexBucket.addSourceJar(SymbolIndexBucket.scala:63)
	at scala.meta.internal.mtags.OnDemandSymbolIndex.$anonfun$addSourceJar$2(OnDemandSymbolIndex.scala:82)
	at scala.meta.internal.mtags.OnDemandSymbolIndex.tryRun(OnDemandSymbolIndex.scala:124)
	at scala.meta.internal.mtags.OnDemandSymbolIndex.addSourceJar(OnDemandSymbolIndex.scala:81)
	at scala.meta.internal.metals.Indexer.addSourceJarSymbols(Indexer.scala:587)
	at scala.meta.internal.metals.Indexer.$anonfun$indexDependencySources$5(Indexer.scala:436)
	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:575)
	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:573)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:933)
	at scala.collection.IterableOps$WithFilter.foreach(Iterable.scala:903)
	at scala.meta.internal.metals.Indexer.$anonfun$indexDependencySources$1(Indexer.scala:427)
	at scala.meta.internal.metals.Indexer.$anonfun$indexDependencySources$1$adapted(Indexer.scala:426)
	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:575)
	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:573)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:933)
	at scala.meta.internal.metals.Indexer.indexDependencySources(Indexer.scala:426)
	at scala.meta.internal.metals.Indexer.$anonfun$indexWorkspace$21(Indexer.scala:346)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.meta.internal.metals.TimerProvider.timedThunk(TimerProvider.scala:25)
	at scala.meta.internal.metals.Indexer.$anonfun$indexWorkspace$20(Indexer.scala:339)
	at scala.meta.internal.metals.Indexer.$anonfun$indexWorkspace$20$adapted(Indexer.scala:335)
	at scala.collection.immutable.List.foreach(List.scala:333)
	at scala.meta.internal.metals.Indexer.indexWorkspace(Indexer.scala:335)
	at scala.meta.internal.metals.Indexer.$anonfun$profiledIndexWorkspace$2(Indexer.scala:143)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.meta.internal.metals.TimerProvider.timedThunk(TimerProvider.scala:25)
	at scala.meta.internal.metals.Indexer.$anonfun$profiledIndexWorkspace$1(Indexer.scala:143)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:678)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

2022.11.03 10:20:17 INFO  time: indexed workspace in 7.22s
2022.11.03 10:20:17 INFO  compiling root (1 scala source)
2022.11.03 10:20:17 INFO  time: compiled root in 0.45s
2022.11.03 10:20:17 INFO  compiling root-test (1 scala source)
2022.11.03 10:20:19 INFO  time: compiled root-test in 0.98s
```

### Operating system

Windows

### Editor/Extension

VS Code

### Version of Metals

0.11.9

**Comment by tgodzik** [comment]
Actually it looks like the same issue as https://github.com/scalameta/metals/issues/4570

We did a workaround there, which should be available in a snapshot already. Could you check it out again?

**Comment by tgodzik** [comment]
That would be `0.11.9+91-762cdcff-SNAPSHOT`

**Comment by AndreuCodina** [comment]
> Actually it looks like the same issue as #4570
> 
> We did a workaround there, which should be available in a snapshot already. Could you check it out again?

The version `0.11.9+91-762cdcff-SNAPSHOT` doesn't solve the problem.

This is the log of VSCode with Metals when I create a new project and import the build with that version (you can see Andreu%2520Codina):

```
Java home: C:\Users\Andreu Codina\AppData\Local\Coursier\cache\arc\https\github.com\AdoptOpenJDK\openjdk8-binaries\releases\download\jdk8u292-b10\OpenJDK8U-jdk_x64_windows_hotspot_8u292b10.zip\jdk8u292-b10
Metals version: 0.11.9+91-762cdcff-SNAPSHOT
2022.11.03 13:38:45 INFO  logging to file C:\git\scala11\.metals\metals.log
2022.11.03 13:38:45 INFO  Started: Metals version 0.11.9+91-762cdcff-SNAPSHOT in workspace 'C:\git\scala11' for client Visual Studio Code 1.72.2.
2022.11.03 13:38:47 INFO  time: initialize in 1.6s
Initializing Scala Debugger
Release notes was not shown: not showing release notes since they've already been seen for your current version
2022.11.03 13:38:47 WARN  Build server is not auto-connectable.
2022.11.03 13:38:47 WARN  no build tool detected in workspace 'C:\git\scala11'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. 
2022.11.03 13:39:02 INFO  
2022.11.03 13:39:02 INFO  Template applied in C:\git\.\scala11
2022.11.03 13:39:02 INFO  
2022.11.03 13:39:02 INFO  time: ran 'giter8.Giter8' in 5.34s
2022.11.03 13:39:22 WARN  no build target for: C:\git\scala11\src\main\scala\Main.scala
2022.11.03 13:39:27 INFO  no build target found for C:\git\scala11\src\main\scala\Main.scala. Using presentation compiler with project's scala-library version: 3.2.1
2022.11.03 13:39:27 INFO  running 'C:\Users\Andreu Codina\AppData\Local\Coursier\cache\arc\https\github.com\AdoptOpenJDK\openjdk8-binaries\releases\download\jdk8u292-b10\OpenJDK8U-jdk_x64_windows_hotspot_8u292b10.zip\jdk8u292-b10\bin\java -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -jar C:\Users\ANDREU~1\AppData\Local\Temp\metals3664175335017942002\sbt-launch.jar -Dbloop.export-jar-classifiers=sources bloopInstall'
2022.11.03 13:39:29 INFO  [info] welcome to sbt 1.7.1 (AdoptOpenJDK Java 1.8.0_292)
2022.11.03 13:39:30 INFO  time: code lens generation in 2.72s
2022.11.03 13:39:30 INFO  time: code lens generation in 2.72s
2022.11.03 13:39:30 INFO  [info] loading global plugins from C:\Users\Andreu Codina\.sbt\1.0\plugins
2022.11.03 13:39:34 INFO  [info] loading settings for project scala11-build-build from metals.sbt ...
2022.11.03 13:39:34 INFO  [info] loading project definition from C:\git\scala11\project\project
2022.11.03 13:39:36 INFO  [info] loading settings for project scala11-build from metals.sbt ...
2022.11.03 13:39:36 INFO  [info] loading project definition from C:\git\scala11\project
2022.11.03 13:39:39 INFO  [success] Generated .bloop\scala11-build.json
2022.11.03 13:39:39 INFO  [success] Total time: 2 s, completed Nov 3, 2022 1:39:39 PM
2022.11.03 13:39:40 INFO  [info] loading settings for project root from build.sbt ...
2022.11.03 13:39:40 INFO  [info] set current project to scala11 (in build file:/C:/git/scala11/)
2022.11.03 13:39:41 INFO  [success] Generated .bloop\root.json
2022.11.03 13:39:41 INFO  [success] Generated .bloop\root-test.json
2022.11.03 13:39:41 INFO  [success] Total time: 1 s, completed Nov 3, 2022 1:39:41 PM
2022.11.03 13:39:41 INFO  time: ran 'sbt bloopInstall' in 14s
2022.11.03 13:39:41 INFO  Attempting to connect to the build server...
2022.11.03 13:39:41 INFO  Bloop uses C:\Users\Andreu Codina\AppData\Local\Coursier\cache\arc\https\github.com\AdoptOpenJDK\openjdk8-binaries\releases\download\jdk8u292-b10\OpenJDK8U-jdk_x64_windows_hotspot_8u292b10.zip\jdk8u292-b10 defined at C:\Users\Andreu Codina\.bloop\bloop.json
2022.11.03 13:39:42 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\git\scala11\.metals\bsp.trace.json or C:\Users\Andreu Codina\AppData\Local\scalameta\metals\cache\bsp.trace.json
2022.11.03 13:39:42 INFO  Attempting to connect to the build server...
2022.11.03 13:39:42 INFO  Bloop uses C:\Users\Andreu Codina\AppData\Local\Coursier\cache\arc\https\github.com\AdoptOpenJDK\openjdk8-binaries\releases\download\jdk8u292-b10\OpenJDK8U-jdk_x64_windows_hotspot_8u292b10.zip\jdk8u292-b10 defined at C:\Users\Andreu Codina\.bloop\bloop.json
2022.11.03 13:39:42 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\git\scala11\project\.metals\bsp.trace.json or C:\Users\Andreu Codina\AppData\Local\scalameta\metals\cache\bsp.trace.json
2022.11.03 13:39:42 INFO  time: Connected to build server in 1.35s
2022.11.03 13:39:42 INFO  Connected to Build server: Bloop v1.5.4
2022.11.03 13:39:44 ERROR issues while parsing: C:\Users\Andreu Codina\AppData\Local\Coursier\cache\v1\https\repo1.maven.org\maven2\org\scala-lang\scala3-library_3\3.2.0\scala3-library_3-3.2.0-sources.jarjava.nio.file.InvalidPathException: Illegal char <:> at index 3: jar:file:///C:/Users/Andreu%2520Codina/AppData/Local/Coursier/cache/v1/https/repo1.maven.org/maven2/org/scala-lang/scala3-library_3/3.2.0/scala3-library_3-3.2.0-sources.jar!/scala/Tuple.scala
	at sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
	at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
	at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
	at sun.nio.fs.WindowsPath.parse(WindowsPath.java:94)
	at sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:255)
	at java.nio.file.Paths.get(Paths.java:84)
	at scala.meta.internal.mtags.CommonMtagsEnrichments$XtensionInputVirtual.$anonfun$filename$2(CommonMtagsEnrichments.scala:337)
	at scala.util.Failure.getOrElse(Try.scala:219)
	at scala.meta.internal.mtags.CommonMtagsEnrichments$XtensionInputVirtual.filename(CommonMtagsEnrichments.scala:337)
	at scala.meta.internal.mtags.ScalaToplevelMtags.loop(ScalaToplevelMtags.scala:203)
	at scala.meta.internal.mtags.ScalaToplevelMtags.indexRoot(ScalaToplevelMtags.scala:50)
	at scala.meta.internal.mtags.MtagsIndexer.index(MtagsIndexer.scala:19)
	at scala.meta.internal.mtags.MtagsIndexer.index$(MtagsIndexer.scala:18)
	at scala.meta.internal.mtags.ScalaToplevelMtags.index(ScalaToplevelMtags.scala:40)
	at scala.meta.internal.mtags.Mtags.toplevels(Mtags.scala:32)
	at scala.meta.internal.mtags.SymbolIndexBucket.indexSource(SymbolIndexBucket.scala:128)
	at scala.meta.internal.mtags.SymbolIndexBucket.addSourceFile(SymbolIndexBucket.scala:105)
	at scala.meta.internal.mtags.SymbolIndexBucket.$anonfun$addSourceJar$2(SymbolIndexBucket.scala:67)
	at scala.collection.immutable.List.flatMap(List.scala:293)
	at scala.meta.internal.mtags.SymbolIndexBucket.$anonfun$addSourceJar$1(SymbolIndexBucket.scala:65)
	at scala.meta.internal.io.PlatformFileIO$.withJarFileSystem(PlatformFileIO.scala:87)
	at scala.meta.internal.io.FileIO$.withJarFileSystem(FileIO.scala:44)
	at scala.meta.internal.mtags.SymbolIndexBucket.addSourceJar(SymbolIndexBucket.scala:63)
	at scala.meta.internal.mtags.OnDemandSymbolIndex.$anonfun$addSourceJar$2(OnDemandSymbolIndex.scala:83)
	at scala.meta.internal.mtags.OnDemandSymbolIndex.tryRun(OnDemandSymbolIndex.scala:128)
	at scala.meta.internal.mtags.OnDemandSymbolIndex.addSourceJar(OnDemandSymbolIndex.scala:82)
	at scala.meta.internal.metals.Indexer.addSourceJarSymbols(Indexer.scala:588)
	at scala.meta.internal.metals.Indexer.$anonfun$indexDependencySources$5(Indexer.scala:437)
	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:575)
	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:573)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:933)
	at scala.collection.IterableOps$WithFilter.foreach(Iterable.scala:903)
	at scala.meta.internal.metals.Indexer.$anonfun$indexDependencySources$1(Indexer.scala:428)
	at scala.meta.internal.metals.Indexer.$anonfun$indexDependencySources$1$adapted(Indexer.scala:427)
	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:575)
	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:573)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:933)
	at scala.meta.internal.metals.Indexer.indexDependencySources(Indexer.scala:427)
	at scala.meta.internal.metals.Indexer.$anonfun$indexWorkspace$21(Indexer.scala:347)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.meta.internal.metals.TimerProvider.timedThunk(TimerProvider.scala:25)
	at scala.meta.internal.metals.Indexer.$anonfun$indexWorkspace$20(Indexer.scala:340)
	at scala.meta.internal.metals.Indexer.$anonfun$indexWorkspace$20$adapted(Indexer.scala:336)
	at scala.collection.immutable.List.foreach(List.scala:333)
	at scala.meta.internal.metals.Indexer.indexWorkspace(Indexer.scala:336)
	at scala.meta.internal.metals.Indexer.$anonfun$profiledIndexWorkspace$2(Indexer.scala:143)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.meta.internal.metals.TimerProvider.timedThunk(TimerProvider.scala:25)
	at scala.meta.internal.metals.Indexer.$anonfun$profiledIndexWorkspace$1(Indexer.scala:143)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:678)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

2022.11.03 13:39:47 INFO  time: indexed workspace in 4.5s
2022.11.03 13:39:47 INFO  compiling root (1 scala source)
2022.11.03 13:39:47 INFO  compiling root-test (1 scala source)
2022.11.03 13:39:47 INFO  time: compiled root in 0.73s
2022.11.03 13:39:49 INFO  time: compiled root-test in 0.79s
```

**Comment by tgodzik** [comment]
Possibly fixed by https://github.com/scalameta/metals/pull/4899 but I need to confirm later on after it's merged. Unless anyway will be able to do it first?

**Comment by AndreuCodina** [comment]
Hello @tgodzik
What backend version number do I have to test?

**Comment by ckipp01** [comment]
> What backend version number do I have to test?

Wait for #4899 to be merged in, and then you can try the snapshot it produces.

**Comment by tgodzik** [comment]
You should be able to test out the newest snapshot

**Comment by AndreuCodina** [comment]
Solved with 0.11.10+105-94ac45a1-SNAPSHOT
Great work, thanks! :)
